src/liballoc/rc.rs

```diff
@@ -570,16 +681,16 @@ impl<T: ?Sized + PartialEq> PartialEq for Rc<T> {
     ///
     /// let five = Rc::new(5);
     ///
-    /// five == Rc::new(5);
+    /// assert!(five == Rc::new(5));
```

Comment:
I went with `==` because we're talking about the definition of `==`. Of course, `assert_eq!` uses `==` but it's an extra layer of indirection.